apps/nextjs-website/src/lib/cmsApi.ts

```diff
@@ -103,6 +106,19 @@ export const getTutorialsProps = async (productSlug?: string) => {
   }
 };
 
+export const getTutorialListPagesProps = async () => {
+  const {
+    config: { FETCH_FROM_STRAPI: fetchFromStrapi },
+  } = buildEnv;
+
+  if (fetchFromStrapi) {
+    const strapiTutorialList = await fetchTutorialList(buildEnv);
```

Comment:
Considering that `makeTutorialListPagesProps` inputs are lists of pages (`strapiTutorialListPages` and `staticTutorialListPages`) and not lists of tutorials, maybe you should rename this constant as follows:
```suggestion
    const strapiTutorialListPages = await fetchTutorialList(buildEnv);
```
and rename tutorialLists constant as well.

I would also consider renaming `fetchTutorialList` method in api.ts adopting the same logic.